Userland/Libraries/LibPDF/Filter.cpp

```diff
@@ -86,59 +86,64 @@ PDFErrorOr<ByteBuffer> Filter::decode_ascii_hex(ReadonlyBytes bytes)
     return output;
 }
 
+static bool is_pdf_whitespace(u8 byte)
```

Comment:
This should probably live in a more common place. But I'm not familiar enough with LibPDF to know where.

Reply:
This is basically https://github.com/SerenityOS/serenity/blob/f7874d03fcdbbc861dd3e7f326716af73cbcb06a/Userland/Libraries/LibPDF/Reader.cpp#L19

How about adding static `is_eol(char)` and `is_whitespace(char)` methods to reader, making `Reader::matches_whitespace()` look like `return !done() && is_whitespace(peek())` (and same for `is_eol()` and calling `Reader::is_whitespace(char)` here?